src/e3/testsuite/__init__.py

```diff
@@ -529,18 +564,26 @@ class Testsuite(TestsuiteCore):
     """
 
     CROSS_SUPPORT = False
-    # set CROSS_SUPPORT to true if the driver should accept --target, --build
-    # --host switches
+    """
```

Comment:
Strange to replace comments by docstring here. The docstring is just ignored

Reply:
As discussed live: this is allowed by PEP257 (https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring)